### PR TITLE
fixing intsights

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready/In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/14555

## Description
Fixing intsights integration for Adidad POC

## Screenshots
Remediation fix 
image

Mail fix:
image

Severity fix:
image

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by ronykoz** [comment]
Still have to go over AskTheAnalyst(Lack of example to work with) and the scoring for Dbot(not sure why do you want to make that change for Dbot @guydemi )

File: Integrations/integration-IntSights.yml [hunk 1]
```diff
@@ -123,6 +123,23 @@ script:
...
+            mails.append(reply.get('Email'))
+
+        return '\n'.join(mails)
+
+
+    def extract_remediation(remidiations):
+        remdies = []
+        string_format = "{0} - Status: {1}"
+        for remdy in remidiations:
+            remdies.append(string_format.format(remdy.get('Value'), remdy.get('Status')))
```
Comment (Itay4) [inline Integrations/integration-IntSights.yml]: If `Value` or `Status` keys will not be in `remdy`, the output would be `None - Status: None`. Can we handle that better?

File: Integrations/integration-IntSights.yml [hunk 2]
```diff
@@ -462,29 +491,65 @@ script:
             'LastSeen': demisto.get(r,'LastSeen'),
             'Domain': demisto.get(r,'Domain'),
             'Status': demisto.get(r,'Status'),
-            'Severity': demisto.get(r,'Severity'),
+            'Severity': demisto.get(r,'Severity').get('Value', 'None'),
```
Comment (Itay4) [inline Integrations/integration-IntSights.yml]: Why did you set the default to be the string `'None'`?

File: Integrations/integration-IntSights.yml [hunk 3]
```diff
@@ -462,29 +491,65 @@ script:
...
             'SourceName': demisto.get(r, 'Source.Name'),
             'SourceConfidence': demisto.get(r, 'Source.Confidence'),
             'IsInAlexa': demisto.get(r, 'Flags.IsInAlexa'),
             'Enrichment Status': demisto.get(r, 'Enrichment.Status'),
             'Enrichment Date': demisto.get(r, 'Enrichment.Data')
         }
-        return ioc_context, ioc_readable
+        dbot_score = {
+            'Indicator': ioc_context['Value'],
+            'Type': ioc_context['Type'],
```
Comment (Itay4) [inline Integrations/integration-IntSights.yml]: What are the possible values for `ioc_context['Type']` ?

File: Integrations/integration-IntSights.yml (hunk 1 above)
Comment (Itay4) [reply Integrations/integration-IntSights.yml]: 👍

File: Integrations/integration-IntSights.yml (hunk 3 above)
Comment (Itay4) [reply Integrations/integration-IntSights.yml]: Yes, should be one of: `ip`, `url`, `domain`, `hash`

File: Integrations/integration-IntSights.yml [hunk 4]
```diff
@@ -1091,6 +1089,51 @@ script:
...
+    - contextPath: File.MD5
+      description: Bad hash MD5
+      type: string
+    - contextPath: File.SHA1
+      description: Bad hash SHA1
+      type: string
+    - contextPath: File.SHA256
+      description: Bad hash SHA256
+      type: string
+    - contextPath: URL.Address
```
Comment (Itay4) [inline Integrations/integration-IntSights.yml]: should be `URL.Data`

File: Tests/conf.json [hunk 5]
```diff
@@ -109,10 +109,6 @@
             ],
             "playbookID": "blockip_test_playbook"
         },
-        {
```
Comment (ronykoz) [inline Tests/conf.json]: Someone duplicated the test for some reason, so i removed the second occurrence

**Comment by guydemi** [comment]
@ronykoz did you manage to get an indicator from IntSights show in the Indicators Page?

**Comment by ronykoz** [comment]
@guydemi yeah attaching a screenshot, seems ok, right?(The Good reputation is before the scoring change to what you said)

screen shot 2018-12-16 at 17 41 44

**Pull request merged**